themes/bootstrap3/less/components/sidebar.less

```diff
@@ -77,9 +78,12 @@
   border: 1px solid @list-group-border;
 }
 
+.checkboxFilter {
```

Comment:
Having a .checkboxFilter class with a different meaning than .checkbox-filter is a bit confusing. Can we name things better here?

Also, I notice that the new styling formats checkbox filters significantly differently, even in the existing SideFacets area...

This branch:

![image](https://user-images.githubusercontent.com/309069/50294407-d686f700-0443-11e9-915a-a31ef64ccb92.png)

Master branch:

![image](https://user-images.githubusercontent.com/309069/50294465-f3bbc580-0443-11e9-8b56-901ecca0f054.png)

Perhaps it is beneficial to have the checkbox filters better aligned with the other facets, but it also seems to take up more vertical space than it used to. Not sure if we should go with this or try to stick with past appearances. @crhallberg, any thoughts?

Reply:
Didn't realize this was waiting on me. I tweaked the padding to remove this discrepancy and to fix having more padding than necessary around the sidebar.

Reply:
Thanks, this looks much better now. There's still a bit more padding in the new styling, but I think it looks more consistent this way, so no further objections. I'll resolve this thread.